docs/performance_archive/granular_tests/testEnterTimeReal_PERFORMANCE

```diff
@@ -124,3 +124,22 @@
 2021-02-11	numberThreads: 5	numberRequests: 200	time: 136
 2021-02-11	numberThreads: 5	numberRequests: 200	time: 19
 2021-02-11	numberThreads: 5	numberRequests: 200	time: 201
+2021-02-11	numberThreads: 5	numberRequests: 200	time: 255
+2021-02-11	numberThreads: 5	numberRequests: 200	time: 402
+2021-02-11	numberThreads: 5	numberRequests: 200	time: 14
+2021-02-11	numberThreads: 5	numberRequests: 200	time: 14
+2021-02-11	numberThreads: 5	numberRequests: 200	time: 15
+2021-02-11	numberThreads: 5	numberRequests: 200	time: 14
+2021-02-11	numberThreads: 5	numberRequests: 200	time: 39
+2021-02-11	numberThreads: 5	numberRequests: 200	time: 47
+2021-02-11	numberThreads: 5	numberRequests: 200	time: 16
+2021-02-11	numberThreads: 5	numberRequests: 200	time: 13
+2021-02-11	numberThreads: 5	numberRequests: 200	time: 18
+2021-02-12	numberThreads: 5	numberRequests: 200	time: 16
+2021-02-12	numberThreads: 5	numberRequests: 200	time: 523
```

Comment:
We had to add locking to prevent corruption at a point, this is where you can see the performance numbers do a nosedive.